src/components/Wallet/WalletControl.jsx

```diff
@@ -57,8 +57,7 @@ class WalletControl extends React.Component {
             coinSelection={naiveCoinSelection}
           />
         );
-      if (walletMode === WALLET_MODES.VIEW)
-        return <NodeSet addNode={addNode} updateNode={updateNode} />;
+      if (walletMode === WALLET_MODES.VIEW) return <SlicesTableContainer />;
```

Comment:
Note that the container doesn't need to be passed anything anymore. It's connected to the redux store and so can manage the rest itself. This makes components more self-contained too.